src/renderer/theme.store.ts

```diff
@@ -26,6 +26,12 @@ import logger from "../main/logger";
 
 export type ThemeId = string;
 
+export enum MonacoTheme {
+  HC_BLACK = "hc-black",
```

Comment:
If there is no way to specify `MonacoTheme.HC_BLACK` as of yet I don't think it should be an option here.

BTW is this a fixed list or a dynamic list?

Reply:
https://github.com/react-monaco-editor/react-monaco-editor/blob/bccb619c3ec5037bdda33afc268a616879334bfe/src/types.ts#L48

It is a fixed list of built-in themes

Reply:
How about adding the checkbox in preferences like "Use Hight contrast editor theme (applicable for Dark theme)"

Reply:
![image](https://user-images.githubusercontent.com/53330707/123378535-f1e79580-d59d-11eb-81d4-2251206ab2b1.png)

HC BLACK

Reply:
Good to know. I think user preferences should be left to a separate PR